R/repr_recordedplot.r

```diff
@@ -1,5 +1,13 @@
 CAIRO_INSTALLED <- requireNamespace('Cairo')
 
+check_capability <- function(dev){
```

Comment:
`){` → `) {`

also can you please put a comment that this is for short-circuiting the slow X11 check? else we’ll probably revert it for some more elegant code some day.